extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -1,14 +1,16 @@
 <?php
 class WikiaRssExternalController extends WikiaController {
 
+	const CACHE_TIME = 3600;
```

Comment:
In practice we should never use double cache time. Longer cache time is only used while other process is computing data to be cached. I don't expect it taking more than few seconds. I used this method to limit number of requests to RSS feed on popular page when cache expires. (We can have many concurrent requests fetching the same content in the same time)